### ISSUE TITLE
Blockchain social networking app

### ISSUE THREAD

**Issue by gupadhyaya** [harmony-one/hackathon#5]
Replicate the Dapp-university's popular blockchain social networking using smart contracts app: https://github.com/dappuniversity/social-network

Deliverables:
* Fork the https://github.com/dappuniversity/social-network repo and submit your fork with changes to make it work on harmony
* Add a README.md file with instructions 
* Video tutorial with instructions on deploying this app and about the app itself

Reward
* 500 ONE w/o video
* 1000 ONE w/ video

**Comment by vikramIde** [comment]
I am taking this next

**Comment by vikramIde** [comment]
It's done

https://github.com/vikramIde/social-network/commit/ccfe230cb0a0a76d26875cae29bf5243e649d0ca

Will share the video next

**Comment by vikramIde** [comment]
Guys Ill be sharing the PR today, Sorry about the late response was stuck with some work

**Comment by vikramIde** [comment]
@gupadhyaya @givp 

May I please! know how to provide PR for these projects. 

As you can see I forked from DAPP UNI directly 

https://github.com/vikramIde/social-network

So I would like to know if @gupadhyaya would fork this to harmony from my account or I have to do some other ways ?

Regards

**Comment by givp** [comment]
@vikramIde Yes, once reviewed we will fork it from your repo into the Harmony account. I'll review shortly. Thank you.

**Comment by vikramIde** [comment]
> @vikramIde Yes, once reviewed we will fork it from your repo into the Harmony account. I'll review shortly. Thank you.

Sure I have finished this one along with video please do review @givp 

https://github.com/vikramIde/social-network

my $ONE address : one15ad34lhszdnz7n8uvnmuzd3w0ay3vy8fdt3343

**Comment by vikramIde** [comment]
https://github.com/harmony-one/dvideo/pull/1

PR done
